src/screens/NoteAdding/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from "react";
+import { API, Storage } from "aws-amplify";
+import { FormGroup, FormControl, ControlLabel } from 'react-bootstrap';
+import './index.css';
+import LoaderButton from '../../components/LoaderButton';
+import config from '../../config';
+import { s3Upload } from '../../libs/awsLib';
+
+class NoteAdding extends Component {
+  constructor(props) {
+    super(props);
+
+    this.file = null;
+    this.state = {
+      isAdding: null,
+      content: ""
+    };
+  }
+
+  async componentDidMount() {
```

Comment:
Why do we need async for this hook?

Reply:
We don't call any async function so no need to use async keyword. However, the function does not have body -> I removed it.